server/node.go

```diff
@@ -170,6 +170,12 @@ func NewNode(ctx storage.StoreContext) *Node {
 	}
 }
 
+// context returns a base context for Node-level use. It is shared with
+// the local sender and contains NodeID and ClusterID (once known).
+func (n *Node) context() context.Context {
```

Comment:
Yeah, once we pass that extra information that will happen. I did not want to mess with the KVSender interface in this change. Or do we not care about the local sender implementing it?